docs/services/navigation/_index.md

```diff
@@ -115,6 +119,54 @@ The following attributes are available for `Navigation` services:
 | `meters_per_sec` | float | Optional | The default linear velocity for the [base](/components/base/) in meters per second. <br> Default: `0.3` |
 | `obstacles` | obj | Optional | Any obstacles you wish to add to the robot's path. See the [Motion Service](/services/motion/) for more information. |
 
+### Configure and calibrate the Frame System Service for GPS Navigation
+
+{{% alert title="Info" color="info" %}}
+
+The [Frame System Service](/services/frame-system/) is an internally managed and mostly static system for storing the "reference frame" of each component of a robot within a coordinate system configured by the user.
+
+It stores the required contextual information for Viam's services like [Motion](/services/motion/) and [Vision](/services/vision/) to use the position and orientation readings returned by components like [movement sensors](/components/movement-sensor/).
+
+{{% /alert %}}
+
+For the most accurate autonomous GPS navigation with the Navigation Service, configure and calibrate the Frame System Service for the components of your robot.
```

Comment:
What happens if you don't configure a frame system? Can you even use the nav service effectively at all?

Reply:
If you happen to orient your movement sensor on your base in the way we expect, then it will work. But if you need to calibrate, then you must configure a frame system.

Reply:
Should we say "For the most accurate..." or is there a way to just tell everyone to do it, unless they orient it way we expect? What are the odds they do

Reply:
We should tell everyone to do it. I think 1 in 4?

Reply:
Cool, done